conda-store-server/conda_store_server/orm.py

```diff
@@ -89,6 +89,37 @@ def validate_role(self, key, role):
         return role
 
 
+class NamespaceRoleMappingV2(Base):
+    """Mapping between roles and namespaces"""
+
+    __tablename__ = "namespace_role_mapping_v2"
+
+    id = Column(Integer, primary_key=True)
+    # Provides access to this namespace
+    namespace_id = Column(Integer, ForeignKey("namespace.id"), nullable=False)
+    namespace = relationship(Namespace, foreign_keys=[namespace_id])
+
+    # ... for other namespace
+    other_namespace_id = Column(Integer, ForeignKey("namespace.id"), nullable=False)
```

Comment:
As mentioned above, these names don't make it clear which namespace has permissions to which.

Reply:
Addressed in another comment above.